src/main/java/edu/iis/powp/decorator/gui/events/RunCommandListener.java

```diff
@@ -0,0 +1,26 @@
+package edu.iis.powp.decorator.gui.events;
+
+import edu.iis.powp.app.DriverManager;
+import edu.iis.powp.command.IPlotterCommand;
+import edu.iis.powp.command.manager.PlotterCommandManager;
+import edu.iis.powp.features.CommandsFeature;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class RunCommandListener implements ActionListener {
```

Comment:
We cannot reuse SelectRunCurrentCommandOptionListener because it is placed in test sources root and it's unavailable from context of CommandTransformerCreatorWindow class during compilation.

Reply:
I mean, that I allow you to move it or use implementation from src/main/java